gpytorch/lazy/lazy_evaluated_kernel_tensor.py

```diff
@@ -15,15 +16,13 @@
 
 
 class LazyEvaluatedKernelTensor(LazyTensor):
-    def __init__(self, kernel, x1, x2, batch_dims=None, squeeze_row=False, squeeze_col=False, **params):
+    def __init__(self, kernel, x1, x2, batch_dims=None, **params):
         super(LazyEvaluatedKernelTensor, self).__init__(kernel, x1, x2, **params)
         self.kernel = kernel
         self.x1 = x1
         self.x2 = x2
         self.batch_dims = batch_dims
-        self.squeeze_row = squeeze_row
-        self.squeeze_col = squeeze_col
-        self.is_batch = self.x1.ndimension() == 3 or (self.x1.ndimension() == 2 and self.squeeze_row)
+        self.is_batch = self.x1.ndimension() == 3
```

Comment:
Should we be checking for > 2 now since presumably we want to start moving the rest of the package to supporting multiple batch dimensions?

Reply:
This particular one is fixed now